src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -22,31 +22,38 @@
 
 //	Bookmarks of places to mine. Add additional bookmarks separated by comma.
 string[] SetMiningSiteBookmark = new[] {
-	"mining_site_bookmark_name",
+	"Asteroid Belts",
```

Comment:
To me it looks like it implies that it only works with asteroid belts. Why change this change of scope?